.changeset/unlucky-kiwis-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': minor
+---
+
+Introducing env prop to have configurable authentication env
```

Comment:
This is the kind of PR that warrants a longer, nice description here. It would probably mention the full pathname of the property like `auth.environment`, that it commonly has the values `development` and `production`, what its default value is, and what the user can do to migrate to use the new flag properly.